GVFS/GVFS.FunctionalTests/Tests/GitCommands/UpdateIndexTests.cs

```diff
@@ -53,7 +52,11 @@ public void UpdateIndexRemoveAddFileOpenForWrite()
             this.FilesShouldMatchCheckoutOfTargetBranch();
 
             // Open Test_ConflictTests/AddedFiles/AddedByBothDifferentContent.txt for write so that it's added to the modified paths database
-            using (FileStream stream = File.Open(Path.Combine(this.Enlistment.RepoRoot, @"Test_ConflictTests\AddedFiles\AddedByBothDifferentContent.txt"), FileMode.Open, FileAccess.Write))
+            using (
+                FileStream stream = File.Open(
+                    Path.Combine(this.Enlistment.RepoRoot, "Test_ConflictTests", "AddedFiles", "AddedByBothDifferentContent.txt"), 
+                    FileMode.Open, 
+                    FileAccess.Write))
             {
                 // TODO 940287: Remove this File.Open once update-index --add\--remove are working as expected

```

Comment:
While we are here, is this TODO still accurate?

Reply:
Let me give it a try with the `File.Open` removed.  Based on the description in the issue I'd guess it's still needed, but it's worth giving it a shot with it removed:

> On a regular git enlistment on NTFS git update-index --remove <file path> will only remove the file from the path if that file is not on disk.

> However, git.exe does not check if the file is on disk if the skip-worktree bit is set.  As a result, on GVFS, git update-index --remove <projected file path> will always remove the file from the path, and this differs from the behavior of a vanilla enlistment on NTFS.

Reply:
Thanks for pointing this out, I was able to remove the `File.Open` as it's no longer needed.  I'm also looking into enabling more of these tests.